### DISCUSSION TITLE
log-transformation or not (two different statements)

### DISCUSSION THREAD

**Discussion by jacobHog** [omnideconv/immunedeconv#161] [Q&A]
Hello,

Thank you for developing and maintaining immunedeconv. 

I need some clarification because when I read the vignette I read that the input file has to be TPM-normalized and log transformed (https://omnideconv.org/immunedeconv/reference/deconvolute_base_custom.html). However, when reading the 'getting started' (https://omnideconv.org/immunedeconv/articles/immunedeconv.html) it says that the input file has to be TPM-normalized and NOT log transformed. 

Which is true?

Thanks in advance

**Comment by mhurtado13** [comment]
Same question here! Can someone clarify this? Thanks!

**Comment by LorenzoMerotto** [comment]
Generally speaking the input should always be TPM normalized. Some methods (namely mMCPcounter, BASE) require the data to be log scaled. Therefore, in the functions there are parameters to perform the scaling. Maybe the documentation was misleading.

**Reply by mhurtado13** [reply]
Hi Lorenzo thanks for your answer and your great work regarding immunedeconv and omnideconv! Regarding this and based on your experience, do you know if it will affect a lot to log transformed the TPM normalization? I believe that if it is already TPM normalized, you can use it like that or log transformed. Or is there a specific reason why you consider in your documentation to put "not log transformed" Thanks again and congratulations
